aio/deploy/helm-chart/kubernetes-dashboard/templates/ingress.yaml

```diff
@@ -37,6 +37,9 @@ metadata:
 {{ toYaml . | indent 4 }}
 {{- end }}
 spec:
+  {{- if .Values.ingress.class.name }}
```

Comment:
It will break since class is not defined by default. To simplify, maybe className? Moreover, using with avoids duplication.